webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -110,6 +167,33 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
           ]}>
           <Input maxLength={80} />
         </Form.Item>
+        {canModifyBindings && resourceManagers.length > 0 && (
+          <>
+            <Divider />
+            <>Namespace Bindings:</>
```

Comment:
These fragments are unnecessary
```suggestion
            Namespace Bindings:
```

Reply:
```suggestion
       Namespace Bindings
```

Reply:
done!